src/tests/JIT/interpreter/InterpreterTester.cs

```diff
@@ -22,9 +22,9 @@ public static void RunTests()
         string interpreterApp = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Interpreter.dll");
 
         var startInfo = new ProcessStartInfo(Path.Combine(coreRoot, corerun), interpreterApp);
-        startInfo.EnvironmentVariables["DOTNET_AltJitName"] = libInterp;
-        startInfo.EnvironmentVariables["DOTNET_AltJitPath"] = Path.Combine(coreRoot, libInterp);
-        startInfo.EnvironmentVariables["DOTNET_AltJit"] = "RunInterpreterTests";
+        startInfo.EnvironmentVariables["DOTNET_InterpreterName"] = libInterp;
+        startInfo.EnvironmentVariables["DOTNET_InterpreterPath"] = Path.Combine(coreRoot, libInterp);
+        startInfo.EnvironmentVariables["DOTNET_Interpreter"] = "RunInterpreterTests";
```

Comment:
When using the AltJit option I noticed I had problems unless I passed all these flags. Shouldn't as of this change be enough to just pass the `DOTNET_Interpreter`, while the rest could be omitted ?

Reply:
Right, it should be sufficient. Let me get rid of the other env variables setting in the test.

Reply:
Done